### PR TITLE
 Arcconf Plugin: Add 'volume-ident-led-on' support

### PR DESCRIPTION
**Pull request by sharath-t**
implementation for 'volume-ident-led-on' API support

### CODE REVIEW

File: plugin/arcconf/arcconf.py [hunk 1]
```diff
@@ -644,3 +644,161 @@ def volume_delete(self, volume, flags=0):
...
+            arcconf identify <ctrlNo> logicaldrive <ldNo> time 3600
+
+            default led blink time is set to 1 hour
+        """
+        if not volume.plugin_data:
+            raise LsmError(
+                ErrorNumber.INVALID_ARGUMENT,
+                "Illegal input volume argument: missing plugin_data property")
+
+        volume_info = volume.plugin_data
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: `plugin_data` should be string only.

File: plugin/arcconf/arcconf.py [hunk 2]
```diff
@@ -644,3 +644,161 @@ def volume_delete(self, volume, flags=0):
...
+        ctrl_id = str(volume_info['ctrl_id'])
+        volume_id = str(volume_info['ld_id'])
+
+        try:
+            self._arcconf_exec(['IDENTIFY', ctrl_id, 'LOGICALDRIVE', volume_id, 'TIME', '3600'], flag_force=True)
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: The text width should be less than 80.

File: plugin/arcconf/arcconf.py [hunk 3]
```diff
@@ -644,3 +644,161 @@ def volume_delete(self, volume, flags=0):
...
+        except ExecError:
+            raise LsmError(ErrorNumber.PLUGIN_BUG, 'Volume-ident-led-on failed unexpectedly')
```
Comment (cathay4t) [inline plugin/arcconf/arcconf.py]: The text width should be less than 80.

File: plugin/arcconf/arcconf.py (hunk 1 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: done - commit id - 0664bffa9df4adbc740142c814d1c1aeb3e33311

File: plugin/arcconf/arcconf.py (hunk 2 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: done - commit id - 0664bffa9df4adbc740142c814d1c1aeb3e33311

File: plugin/arcconf/arcconf.py (hunk 3 above)
Comment (sharath-t) [reply plugin/arcconf/arcconf.py]: done - commit id - 0664bffa9df4adbc740142c814d1c1aeb3e33311

**Comment by tasleson** [comment]
Integrated into https://github.com/libstorage/libstoragemgmt/pull/386

**Pull request closed without merging**